README.md

```diff
@@ -5,54 +5,35 @@
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](https://webchat.freenode.net/?channels=%23ipfs)
 [![](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
-> Compact self-describing codecs. Save space by using predefined multicodec tables.
+> Save space by using the predefined multicodec table
```

Comment:
This is a *claim*. I think the subtitle should reflect what multicodec *is*. I'd describe it as a canonical table enumerating well-known content types.

Reply:
What about something like "Canonical table of of codecs used by various multiformats". I still don't like that one, if anyone can come up with something better, that would be great.